app/routes/employee.py

```diff
@@ -1,8 +1,8 @@
 from fastapi import APIRouter, status, HTTPException, Path
-from app.dao.dao_employee import createEmployee, getAll, getOne, updateEmployee, updateEmployeeLogin, deleteEmployee
+from app.dao.dao_employee import createEmployee, getAll, getOne, updateEmployee, updateEmployeeLogin, deleteEmployee, getEmployeeInfo, updateEmployeeInfo
 from fastapi.encoders import jsonable_encoder
 from fastapi.responses import JSONResponse
-from app.schemas.employee import Employee, EmployeeUpdate, EmployeeUpdateLogin
+from app.schemas.employee import Employee, EmployeeUpdate, EmployeeUpdateLogin, EmployeeUpdateInfo
```

Comment:
Otimizar importações
Ex: from app.schemas import employee as schemas_employee

Reply:
Oi Nicholas, não entendi muito bem como fazer essa alteração. To recebendo um TypeError aqui:

TypeError: Annotated[t, ...]: t must be a type. Got <module 'app.schemas.employee' from '/Users/marco/Documents/00 Programação/Startfuture/onboarding-.

Reply:
Ao invés de acessar o arquivo e importar cada função que vai usar
Você pode Importar o arquivo em si e dar um "apelido" a ele

Vamos supor que temos o APP
Dentro de APP temos SCHEMAS
E dentro de SCHEMAS temos o arquivo EMPLOYEE com as funções que você precisa

Então usamos:
from app.schemas import employee

Em alguns casos vamos ter outro arquivo com esse mesmo nome 
Pra não dar conflito usamos "as" para dar um apelido a ele

Então ficaria:
from app.schemas import employee as schemas_employee

pra usar alguma coisa desse arquivo, usamos: "nome dele . nome da função"

ex: classe = schemas_employee.Employee